dspace/config/spring/cris/cris-components.xml

```diff
@@ -450,6 +471,8 @@
 			<map>
 				<entry key="selected" value-ref="selectedPublicationForEventComponent" />
 				<entry key="all" value-ref="allPublicationForEventComponent" />
+				<!-- <entry key="direct" value-ref="directPublicationForEventComponent" /> -->
```

Comment:
you can eventually configure a category to show only the publications directly related to the current event and another category with publications related to one of the sub events.

This configuration is commented out because we don't think it makes any sense for the event but could be useful in other parent-child relation (like organisation units)